### PR TITLE
EPMRPP-37698 || Email notifications backup

### PR DESCRIPTION
**Pull request by Yumfriez**

### CODE REVIEW

File: src/main/java/com/epam/ta/reportportal/commons/SendCase.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ * Email notification cases enumerator for project settings
+ *
+ * @author Andrei_Ramanchuk
+ */
+public enum SendCase {
+
+	//@formatter:off
+	ALWAYS("always"),
+	FAILED("failed"),
+	TO_INVESTIGATE("to_investigate"),
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/commons/SendCase.java]: let's make it camel case as it is used by ui

Comment (Yumfriez) [reply src/main/java/com/epam/ta/reportportal/commons/SendCase.java]: done

File: src/main/java/com/epam/ta/reportportal/entity/project/Project.java [hunk 2]
```diff
@@ -62,6 +63,9 @@
 	@OneToMany(mappedBy = "project", cascade = { CascadeType.PERSIST, CascadeType.REMOVE }, fetch = FetchType.LAZY)
 	private Set<ProjectIssueType> projectIssueTypes = Sets.newHashSet();
 
+	@OneToMany(mappedBy = "project", cascade = { CascadeType.PERSIST, CascadeType.REMOVE }, fetch = FetchType.LAZY)
+	private Set<EmailSenderCase> emailCases;
```
[the remark this answers is not in the document]
Comment (Yumfriez) [reply src/main/java/com/epam/ta/reportportal/entity/project/Project.java]: renamed to sender case

File: src/main/java/com/epam/ta/reportportal/entity/project/ProjectUtils.java [hunk 3]
```diff
@@ -89,6 +96,74 @@ private ProjectUtils() {
 		return projectIssueTypes;
 	}
 
+	/**
+	 * Setup default project email configuration
+	 *
+	 * @param project
+	 * @return project object with default email config
+	 */
+	public static Project setDefaultEmailConfiguration(Project project) {
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/entity/project/ProjectUtils.java]: default notification config?

Comment (Yumfriez) [reply src/main/java/com/epam/ta/reportportal/entity/project/ProjectUtils.java]: renamed

**Pull request merged**